data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -14,14 +14,14 @@ mkdir -p "${ARTIFACTS}/bootstrap/containers"
 sudo crictl ps --all --quiet | while read -r container
 do
     container_name="$(sudo crictl ps -a --id "${container}" -v | grep -oP "Name: \\K(.*)")"
-    sudo crictl logs "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}.log"
-    sudo crictl inspect "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}.inspect"
+    sudo crictl logs "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}-${container}.log"
+    sudo crictl inspect "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}-${container}.inspect"
 done
 mkdir -p "${ARTIFACTS}/bootstrap/pods"
 sudo podman ps --all --quiet | while read -r container
 do
-    sudo podman logs "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.log"
-    sudo podman inspect "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.inspect"
+    sudo podman logs "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}-${container}.log"
```

Comment:
This seems off.

Reply:
/hold

Can we drop the `${container}-${container}` doubling here and in `installer-masters-gather.sh `?